src/app/components/data-entry-grid/index.html

```diff
@@ -0,0 +1,232 @@
+			</ul>
+	</sky-demo-page-content>
+
+    </sky-tab>
+
+    <sky-tab tabHeading="Development">
+      <p>
+        The SKY UX ag-Grid module provides styles, a service for default gridOptions, and SKY UX components for cell rendering and editing in ag-Grid. If you have never used ag-Grid before, <a href="https://www.ag-grid.com/angular-getting-started/">the Angular getting started</a> guide covers the basics. For more in depth information about ag-Grid and its capabilities, please reference <a href="https://www.ag-grid.com/documentation-main/documentation.php">the ag-Grid documentation.</a>
+      </p>
+
+      <sky-demo-page-module-info
+        skyuxModule="SkyAgGridModule"
+        packageName="@skyux/ag-grid"
+        githubRepo="skyux-ag-grid"
+      >
+      </sky-demo-page-module-info>
+
+      <sky-demo-page-properties
+      sectionHeading="SkyAgGridService methods"
+      >
+        <p>
+          <sky-code>SkyAgGridService</sky-code> is a service that provides methods to get ag-Grid <sky-code>gridOptions.</sky-code> The <sky-code>gridOptions</sky-code> can be overridden, and include registered SKY UX column types. Use the service when you are using ag-Grid to ensure your grid matches the standard SKY UX functionality.
+        </p>
+        <sky-demo-page-property
+          propertyName="getGridOptions(args: SkyGetGridOptionsArgs)"
+          isOptional="true"
+        >
+          Returns <a href="https://www.ag-grid.com/javascript-grid-properties/">ag-Grid <sky-code>gridOptions</sky-code></a> with default SKY UX options, SKY UX styling, and SKY UX cell renderers registered for read-only grids.
+        </sky-demo-page-property>
+        <sky-demo-page-property
+          propertyName="getEditableGridOptions(args: SkyGetGridOptionsArgs)"
+          isOptional="true"
+        >
+          Returns <a href="https://www.ag-grid.com/javascript-grid-properties/">ag-Grid <sky-code>gridOptions</sky-code></a> with default SKY UX options, SKY UX styling, and SKY UX cell editors registered for editable grids.
+        </sky-demo-page-property>
+      </sky-demo-page-properties>
+
+      <sky-demo-page-properties
+      sectionHeading="SkyGetGridOptionsArgs type"
+      >
+        <sky-demo-page-property
+          propertyName="gridOptions"
+          isOptional="false"
+        >
+          <a href="https://www.ag-grid.com/javascript-grid-properties/">Ag-Grid <sky-code>gridOptions</sky-code></a> to override the default SKY UX gridOptions. The only options that cannot be overridden are the predefined SKY UX column types for components and column <sky-code>cellClassRules</sky-code>, which is used to enforce required cell styling.
+        </sky-demo-page-property>
+        <sky-demo-page-property
+          propertyName="locale"
+          isOptional="true"
+          default="en-us"
+        >
+          The locale used for location-specific formatting, like the displayed date value for the <sky-code>SkyCellType.Date</sky-code>.
```

Comment:
We usually start with a verb so "Specifies the locale used for location-specific formatting such as date values for the `<sky-code>SkyCellType.Data</sky-code>." We also include what type of values it accepts so adding in a "This property accepts `string` values." (It takes strings, yes?)